files/image_config/hostcfgd/hostcfgd

```diff
@@ -292,7 +257,69 @@ class HostConfigDaemon:
                 syslog.syslog(syslog.LOG_WARNING, "Eanble state of feature '{}' is None".format(feature_name))
                 continue
 
-            update_feature_state(feature_name, state, feature_table[feature_name]['has_timer'])
+            if feature_name + '.service' in generated_services_list:
+                self.host_feature_list.append(feature_name)
+
+            if feature_name + '.service' in generated_multi_instance_services_list:
+                self.namespace_feature_list.append(feature_name)
+
+        for feature_name in feature_table.keys():
+            self.update_feature_state(feature_name, state, feature_table[feature_name]['has_timer'])
+
+    def update_feature_state(self, feature_name, state, has_timer):
+        # Create feature name suffix depending feature is running in host or namespace or in both
+        feature_name_suffix_list = (([feature_name] if feature_name in self.host_feature_list else []) +
+                                   ([(feature_name + '@' + str(asic_inst)) for asic_inst in range(device_info.get_num_npus()) 
+                                   if feature_name in self.namespace_feature_list]))
+
+        if not feature_name_suffix_list:
+            syslog.syslog(syslog.LOG_ERR, "Feature '{}' service not available"
+                          .format(feature_name))
+
+        feature_suffixes = ["service"] + (["timer"] if ast.literal_eval(has_timer) else [])
+
+        if state == "enabled":
+            start_cmds = []
+            for feature_name_suffix in feature_name_suffix_list:
+                for suffix in feature_suffixes:
+                    start_cmds.append("sudo systemctl unmask {}.{}".format(feature_name_suffix, suffix))
```

Comment:
That's great work. I know it merged already, but wanted to follow up the story here if we have plan to support enabling/disabling services for particular asics in system?

More specifically, says there are 2 asics in system, we want to disable lldp on asic 1, but not asic 2.

I imagine that to do that, we need 

(1) Create entry like `FEATURE|lldp|1` with `enabled = false` and `FEATURE|lldp|2` with `enabled = true`. hostcfgd listens and reacts to the change. It will turn off `lldp@1.service` but keep `lldp@2.service` on.

Or (2) Create `hostcfgd@1` for asic 1 and `hostcfgd@2` for asic 2. `hostcfgd@1` listens `database@1 CONFIG_DB`, and reacts to `FEATURE|lldp`. Same for `hostcfgd@2` which listens `database@2 CONFIG_DB`. But this one may not work if we want to enable/disable database. If database doesn't exits, `hostcfgd@x` will have nothing to listen and react.